src/DotLiquid/Strainer.cs

```diff
@@ -18,7 +20,8 @@ public class Strainer
     {
         private static readonly Dictionary<string, Type> Filters = new Dictionary<string, Type>();
         private static readonly Dictionary<string, Tuple<object, MethodInfo>> FilterFuncs = new Dictionary<string, Tuple<object, MethodInfo>>();

```

Comment:
Related to comment below - can we replace Tuple with a type with more meaningly properties? Seeing Item1 and Item2 in the code makes it hard to read and understand, particularly when using `var`.